docs/mesos/README.md

```diff
@@ -6,78 +6,66 @@
 > You are viewing the calico-containers documentation for release **release**.
 <!--- end of master only -->
 
-# Mesos with Calico networking
-Calico can be used as a network plugin for Mesos both for the Docker
-Containerizer and the Unified Containerizer.
+# Calico Networking for Mesos
+Calico introduces fine-grained security policies to Mesos, while
```

Comment:
Mention IP per task, highly scalable.  Use of routable IPs negates the need for port-forwarding.